cmd/esdiff/main.go

```diff
@@ -0,0 +1,150 @@
+package main
```

Comment:
If you turn off replication and indexing on the target index until the copy has finished,  it should be at least twice as fast.